### ISSUE TITLE
Energy cards don't work when added to other dashboards when the Overview dashboard is not the default

### ISSUE THREAD

**Issue by rljk99** [home-assistant/frontend#21481]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.
- [X] I have tried reproducing the issue in [safe mode](https://www.home-assistant.io/blog/2023/11/01/release-202311/#restarting-into-safe-mode) to rule out problems with unsupported custom resources.

### Describe the issue you are experiencing

I have added 3 of the energy cards (gauges for "self consumed solay energy", "self sufficiency", and "net energy consumed from the grid") to two of my dashboards.

Initially they seem to work fine, but after I close the browser (end of work session) and then restart the browser and access home-assistant, the cards no longer function.

After some experimentation, I discovered that if I make the Overview dashboard my default, the cards operate correctly. When I switch my default dashboard to something other than "Overview" the cards stop working.

I have been able to duplicate this several times, and the problem exists with chrome and firefox.

### Describe the behavior you expected

I expected the energy cards to display the data regardless of which dashboard is selected as default 

### Steps to reproduce the issue

1.Dashboard other than overview is default
2.Add energy cards listed above to dashboard
3.observe the cards don't display data 
4.make overview dashboard default
5.observe energy cards display data properly (sometimes closing the browser and reopening it is required to get the data to display.

### What version of Home Assistant Core has the issue?

2024.6.1

### In which browser are you experiencing the issue with?

Google Chrome 126.0.6478.185

### Which operating system are you using to run this browser?

Windows 11

**Comment by karwosts** [comment]
Just to confirm, you definitely tested this in safe mode (all custom cards disabled)? There are known custom cards which break energy cards functionality.

**Comment by rljk99** [comment]
Prior to your message, I didn’t know that home assistant had a safe mode. I thought the safe mode question referred to the browser (as in “incognito window” in chrome).

Once I figured out how to get home assistant into safe mode, I found that my problem is in fact caused by one of the custom cards I was using. I was able to trace it to  the “energy-period-selector-plus” card I had found in HACS. I replaced that card with the standard issue “energy-date-selection” energy card and my problem went away.

I apologize for my less than complete troubleshooting, but I learned something today, and that’s never a bad thing.

Thank you for your help.

I believe the issue can be closed
